exercises/collatz-conjecture/collatz-conjecture.spec.js

```diff
@@ -0,0 +1,38 @@
+var collatzConjecture = require('./collatz-conjecture');
```

Comment:
Again, you'll want to use `PascalCase` for the class import: https://github.com/airbnb/javascript#naming--PascalCase